test/tests/ver-1c/ver-1c.i

```diff
@@ -1,7 +1,10 @@
+# Locations for concentration comparison
+# TMAP7 - 12, 0.25, h (10)
+# TMAP4 - 12, 0,    h (10)
 [Mesh]
   type = GeneratedMesh
   dim = 1
-  nx = 1000
+  nx = 1e4
```

Comment:
It still runs in less than 2 seconds with this mesh?

Reply:
2.393 seconds on my mac. Maybe I should make the tests heavy and add a regular exodiff test?

Reply:
No, 2.4 seconds is acceptable. We just need to be mindful not to make tests too computational costly and I just wanted to check that you made sure of that.